src/Protocol/PortableContact.php

```diff
@@ -734,7 +734,7 @@ private static function parseNodeinfo1($nodeinfo_url)
 			}
 		}
 
-		if (is_array($nodeinfo['metadata']) && isset($nodeinfo['metadata']['nodeName'])) {
+		if (!empty($nodeinfo['metadata']) && isset($nodeinfo['metadata']['nodeName'])) {
```

Comment:
Please simplify to `isset($nodeinfo['metadata']['nodeName'])`.

Reply:
I'm not totally sure, but I'm convinced that this "isset" would create a problem when "metadata" isn't present.

Reply:
I just tested in PHP Fiddle, it doesn't complain.
```php
$nodeinfo = [
  'foo' => 'bar'
];

isset($nodeinfo['metadata']); // No Notice

isset($nodeinfo['metadata']['nodeName']); // No Notice
```